server/helpers/cleanupArticlesResponse.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @description function to return well formatted articles
+ * @param {array} rows
+   * @returns {object} returns articles object
+*/
+const cleanupArticlesResponse = (rows) => {
+  const result = [];
+  for (let x = 0; x < rows.length; x += 1) {
+    const {
+      id,
+      title,
+      slug,
+      description,
+      body,
+      rating,
+      createdAt,
+      updatedAt,
+      author,
+      readTime,
+      category,
+      likes,
+      dislikes,
+      comments
+    } = rows[x].dataValues;
+    const newObj = {
+      id,
+      title,
+      slug,
+      description,
+      body,
+      readTime,
+      category: category.categoryName,
+      author: author.fullName,
+      authorAvatar: author.avatarUrl,
```

Comment:
I think you should put the author related attributes into a nested author object here. This would make the response easier to read and look something like [this](https://github.com/andela/valinor-ah-backend#multiple-articles)

Reply:
True. Done.